test.sh

```diff
@@ -88,9 +88,9 @@ function prepare_testdir_under_gopath {
 function generate_crd_resources {
 # Setup env vars
   export PATH=/tmp/kubebuilder/bin/:$PATH
-  export TEST_ASSET_KUBECTL=/tmp/kubebuilder/bin/kubectl
-  export TEST_ASSET_KUBE_APISERVER=/tmp/kubebuilder/bin/kube-apiserver
-  export TEST_ASSET_ETCD=/tmp/kubebuilder/bin/etcd
+#  export TEST_ASSET_KUBECTL=/tmp/kubebuilder/bin/kubectl
+#  export TEST_ASSET_KUBE_APISERVER=/tmp/kubebuilder/bin/kube-apiserver
+#  export TEST_ASSET_ETCD=/tmp/kubebuilder/bin/etcd
 
```

Comment:
Currently, code in the PR is not getting exercised by test.sh because controlpane.Start() will pick these environment variables the first time itself before new code kicks in.

Reply:
Put the TEST_ASSET ENV back.